operations/customers.md

```diff
@@ -593,3 +593,97 @@ Attaches the specified file to the customer profile.
 {}
 ```
 
+## Add registration card
+
+Creates registration card as pdf with received signature and attaches it to the customer profile.
+
+### Request 
+
+`[PlatformAddress]/api/connector/v1/customers/addRegistrationCard`
+
+```javascript
+{
+    "ReservationId": "71C63D19-2CD5-465D-8FC5-AB0400F650A1",
+    "CustomerId": "A30E2AFA-5752-455C-B2F9-AB0400F64EA1",
+    "Customer": {
+        "Gender": "Male",
+        "FirstName": "John",
+        "LastName": "Doe",
+        "BirthDate": "2000-10-10",
+        "Phone": "888888888",
+        "NationalityCode": "US",
+        "Address": {
+            "Line1": "123 Main St",
+            "Line2": "Address line 2",
+            "City": "Anytown",
+            "PostalCode": "888 88",
+            "CountryCode": "AU",
+            "CountrySubdivisionCode": "AU-SA"
+        },
+        "Passport": {
+            "Number": "987654321",
+            "Issuance": "2015-02-15",
+            "Expiration": "2015-12-12",
+            "IssuingCountryCode": "US"
+        }
+    },
+    "Signature": {
+        "MimeType": "image/png",
+        "Base64Data": "iVBORw0KGgoAAAANSUhEUgAAADwAAAAyCAIAAACPlC9VAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAABlSURBVGhD7dKxDcAgEARBmqcXusQOXIJZCWk2u+Q1wY99YdBV0FXQVdBV0FXQVdBV0FXQVdBV0FWn0GutOec3/u4U+hWPceq496iCroKugq6CroKugq6CroKugq6CroKuuhC99wN5AwAIQWG9+wAAAABJRU5ErkJggg=="
+    }
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Customer` | [Customer](customers.md#RegistrationCardCustomer) | required | Details of [Customer](customers.md#RegistrationCardCustomer) |
```

Comment:
If it's a link to `RegistrationCardCustomer`, shouldn't it be named like this, not just `Customer`?